README.md

```diff
@@ -59,6 +59,88 @@ style guide that you will not find in the GitHub one.
 
 ## Standards ##
 
+### Naming Convention ###
+
+With Swift 3, Apple updated the Swift [API design guidelines](https://swift.org/documentation/api-design-guidelines/) to promote clarity over brevity.
+
+Capitalize all type names with camel case, while function and variable names start with a lower case. Avoid acronyms and abbreviations for clarity and readability.
+
+#### Protocols ####
+
+Protocols describing something should be a noun: `Collection`, `Element`. Protocols describing an ability should end with “ing” or “able”: `Evaluatable`, `Printable`, `Formatting`.
+
+#### Enum ####
+
+Enum cases should start with a lower case and be camel case.
+
+```
+enum Color {
+    case red
+    case blue
+    case green
+}
+```
+
+#### Functions ####
+
+Include words that can make the function’s behavior clearer. Example of a function that removes an element at an index x.
+
+**Preferred:**
+```swift
+func remove(at index: Index) -> Element
+```
+
+**Not Preferred:**
+```swift
+func remove(index: Index) -> Element
+```
+
+*Rationale*: It is better to specify that we are removing the element at the given index, and we are not trying to remove the given parameter itself, to make the behavior of the function very clear.
+
+Avoid unnecessary words in the function name.
+
+**Preferred:**
+```swift
+func remove(_ element: Element) -> Element?
+```
+
+**Not Preferred:**
+```swift
+func removeElement(_ element: Element) -> Element?
+```
+
+*Rationale*: Make the code clear and concise whenever possible. Adding extra unnecessary words will make the code harder to read and understand.
+
+Swift 3 encourages to name functions to be read as a sentence according to their side effects.
+
+**Preferred:**
+```swift
+x.insert(y, at: z)          “x, insert y at z”
+x.subViews(havingColor: y)  “x's subviews having color y”
+x.capitalizingNouns()       “x, capitalizing nouns”
+```
+
+**Not Preferred:**
+```swift
+x.insert(y, position: z)
+x.subViews(color: y)
+x.nounCapitalize()
```

Comment:
I get that, was just wondering if we want to mirror the *preferred* section. Since you used `capitalizingNouns` I thought `nounCapitalizing` would be a better counter example. Not blocking though